src/main.h

```diff
@@ -59,4 +77,22 @@ void initializeDisplay();
  */
 void handleTime();
 
+void buttonRead(int buttonPress);
```

Comment:
Can you remove the functions here that we no longer have in main.cpp?

Reply:
Done